extension/src/utilities.rs

```diff
@@ -53,26 +61,47 @@ pub fn generate_periodic_normal_series(
     let periodic_magnitude = periodic_magnitude.unwrap_or(100.0);
     let standard_deviation = standard_deviation.unwrap_or(100.0);
 
-    use rand_distr::Distribution;
     use rand::SeedableRng;
     use rand_chacha::ChaCha12Rng;
+    use rand_distr::Distribution;
 
     let mut rng = match rng_seed {
         Some(v) => ChaCha12Rng::seed_from_u64(v as u64),
-        None => ChaCha12Rng::from_entropy()
+        None => ChaCha12Rng::from_entropy(),
     };
 
     let distribution = rand_distr::Normal::new(0.0, standard_deviation).unwrap();
 
     let series_start: i64 = series_start.into();
-    (0..series_len).step_by(sample_interval as usize).map(move |accum| {
-        let time = series_start + accum;
-        let base = base_value + f64::sin(accum as f64 / (2.0 * std::f64::consts::PI * period as f64)) * periodic_magnitude;
-        let error = distribution.sample(&mut rng);
-        (time.into(), base + error)
-    })
+    (0..series_len)
+        .step_by(sample_interval as usize)
+        .map(move |accum| {
+            let time = series_start + accum;
+            let base = base_value
+                + f64::sin(accum as f64 / (2.0 * std::f64::consts::PI * period as f64))
+                    * periodic_magnitude;
+            let error = distribution.sample(&mut rng);
+            (time.into(), base + error)
+        })
 }
 
+// Returns days in month
+extension_sql!(
+    "\n\
+create or replace function toolkit_experimental.days_in_month(date timestamptz) RETURNS int AS $$\n\
+SELECT CAST(EXTRACT('day' FROM ($1 + interval '1 month' - $1)) as INTEGER)\n\
+$$ LANGUAGE SQL STRICT IMMUTABLE PARALLEL SAFE;\n\
+",
+    name = "days_in_month",
+);
+
+// Normalizes metric based on reference date and days
+extension_sql!("\n\
+create or replace function toolkit_experimental.month_normalize(metric float8, reference_date timestamptz, days float8 DEFAULT 365.25/12) RETURNS float8 AS $$\n\
```

Comment:
@JLockerman Do we need to do something for these in terms dealing with the potential security vulnerability of using CREATE OR REPLACE?

Reply:
@davidkohn88 if it's in experimental I think we can do an unconditional `CREATE`; the experimental schema is always dropped on update so any collision is a bug.

Reply:
What about when we graduate it?

Reply:
But for now that should work...